spec/index.bs

```diff
@@ -3916,15 +3920,27 @@ dictionary GPURasterizationStateDescriptor {
     GPUDepthBias depthBias = 0;
     float depthBiasSlopeScale = 0;
     float depthBiasClamp = 0;
+
+    GPUSize32 sampleCount = 1;
+    GPUSampleMask sampleMask = 0xFFFFFFFF;
+    boolean alphaToCoverageEnabled = false;
```

Comment:
alpha-to-coverage occurs after the fragment stage runs.

Reply:
the order of these fields is not following the pipeline (in the PR, at least). Only the order of top-level fields in `GPURenderPassDescriptor` does

Reply:
they don't follow the pipeline within the sub-descriptor, but if it specifically takes effect in a stage after the rasterization stage, shouldn't it not be inside the rasterization state descriptor?

Reply:
yes, I added this note to the description:
> Note: we probably want to move the alpha-to-coverage after the fragment shader